tests/integration/test_charm.py

```diff
@@ -71,7 +92,8 @@ async def connect_client_to_server(ops_test: OpsTest, application):
         minio_cmd,
     )
 
-    return await ops_test.run(*kubectl_cmd)
```

Comment:
No it worked fine for me (provided microk8s is available).  Was it giving you trouble?

Reply:
Ohhh I see what you mean.  Sorry, I misunderstood.  You're asking why did I change from
```python
    ret_code, stdout, stderr = await ops_test.run(*kubectl_cmd)
```
to
```python
    ret_code, stdout, stderr = await ops_test.run(*kubectl_cmd)
    return ret_code, stdout, stderr
```
(eg: why did I unpack the return from `ops_test.run`)?

It was purely style/readability.  I had condenced things down to just `return await ops_test...`, but when writing the docstring realised returning straight from that function made it non-obvious what actually got returned (especially since it is just a tuple of stuff).  Unpacking and returning felt more readable.  I'm flexible though

Reply:
You are right, it is not so obvious it returns a tuple, we can leave it as it is.